src/ProcessManager/SeleniumManager.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Panthère project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Panthere\ProcessManager;
+
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\WebDriver;
+use Facebook\WebDriver\WebDriverCapabilities;
+
+final class SeleniumManager implements BrowserManagerInterface
+{
+    private const DEFAULT_HOST = 'http://127.0.0.1:4444/wd/hub';
```

Comment:
I'm not even sure we need this constant, can be the default value in the constructor directly.